Frontend/app/secure-test/secure-test.html

```diff
@@ -0,0 +1,7 @@
+<div class="row">
```

Comment:
I'd add those steps you showed me for how to test it as part of this html so its common knowledge